examples/bridge-marker.yml

```diff
@@ -0,0 +1,78 @@
+apiVersion: extensions/v1beta1
```

Comment:
why do we need cluster/examples/bridge-marker.yml and examples/bridge-market.yml? They seem identical except for image repo (?) Should we have some template mechanism to substitute that?

Reply:
examples/bridge-marker.yml is generated. Removing, adding to gitignore

Reply:
there is still cluster/examples/bridge-market.yml and manifests/*.in that seem to be duplicates.

Reply:
Removed both.
To have an example you need to run "make manifest"
sync is using the generated manifest now.